recipe/meta.yaml

```
@@ -52,11 +52,11 @@ outputs:
 
     requirements:
       host:
-        - python >=3.8
+        - python >=3.9
       run:
-        - python >=3.8
+        - python >=3.9
         - openmm >=7.6
-        - openff-interchange =0.2
+        - openff-interchange >=0.3.0
         - rdkit
         - ambertools >=20
```

```suggestion
        - ambertools =22
```
We might not want to let 23 slip in if we haven't tested it

---

Good point, but I'd rather be permissive now and pin later with a new build if a problem arises. Our deployment tests (conda CI) should let us know quick if this is a problem.

---

Can we a least constrain to 22 or newer? 20 allows ParmEd 4 to be brought in, which breaks examples

---

Yes, I'll get that into the next build/release